services/newsfeed/package.json

```diff
@@ -27,16 +27,29 @@
     "eslint": "eslint ./api ./knowledge ./services ./test ./*.js",
     "eslint-fix": "eslint --fix ./api ./knowledge ./services ./test ./*.js",
     "clean": "rm -rf ./node_modules",
-    "watch": "supervisor -w . -i ./node_modules app.js"
+    "watch": "supervisor -w . -i ./node_modules app.js",
+    "test:unit": "./node_modules/.bin/jest --config=jest.config.unit.js --detectOpenHandles --forceExit"
   },
   "dependencies": {
+    "bluebird": "^3.7.2",
+    "knex": "^0.95.4",
+    "mysql": "^2.18.1",
+    "lisk-service-framework": "https://github.com/LiskHQ/lisk-service/raw/0ba4de1b8dc37fd35f38ce122321345284f35771/framework/dist/lisk-service-framework-1.2.1.tgz",
+    "node-cron": "=2.0.3",
     "html-entities": "^2.3.2",
     "html-to-text": "^1.2.1",
-    "lisk-service-framework": "https://github.com/LiskHQ/lisk-service/raw/0ba4de1b8dc37fd35f38ce122321345284f35771/framework/dist/lisk-service-framework-1.2.1.tgz",
+    "await-to-js": "^3.0.0",
+    "debug": "^4.3.1",
+    "log4js": "^6.3.0",
+    "moleculer": "=0.13.12",
     "moment": "^2.29.1",
     "object-hash": "^2.2.0",
-    "stack-trace": "^1.0.0-pre1",
+    "pg-promise": "^10.10.2",
+    "require-all": "^3.0.0",
+    "stack-trace": "0.0.10",
```

Comment:
Please review those dependencies, most of them are not needed. The development-0.5.0 branch has updated package and package-lock files.